identity-service/src/remoteConfig.js

```diff
@@ -0,0 +1,21 @@
+const REMOTE_CONFIG_FEATURE = 'remote_config'
+
+// Declaration of remote config variables set in optimizely
+const VARS = {
+  TRENDING_EXPERIMENT: 'TRENDING_EXPERIMENT'
+}
+
+// Use a dummy user id since remote config is enabled by default
+// for all users
+const DUMMY_USER_ID = 'ANONYMOUS_USER'
+
+const getRemoteVar = (optimizelyClient, variable) => {
```

Comment:
I think we may need to handle the edge case where this is called prior to the optimizely client initting? We had to do that in the client (see `getValueHelper`)

Reply:
i thought about this, but I *really* don't think this is a concern because this gets stood up when the server comes online vs. every session in the client. do you really want to see this?

Reply:
I'm just worried about the case of a few requests getting through right after we startup and before optimizely is initted - I think this would actually cause a crash in node because we'd try to reference the client, which would be undefined bc it's not attached to the app until it inits. If you're confident this won't happen I don't need defaults

Reply:
ill do something super trivial